templates/api/keras_cv/models/index.md

```diff
@@ -0,0 +1,88 @@
+# KerasCV Models
+
+KerasCV contains end-to-end implementations of popular model
+architectures. These models can be created in two ways:
+
+- Through the `from_preset()` constructor, which instantiates an object with
+  a pre-trained configurations, and (optionally) weights.
+  Available preset names are listed on this page.
+
+```python
+model = keras_cv.models.RetinaNet.from_preset(
+    "resnet50_v2_imagenet",
+    num_classes=20,
+    bounding_box_format="xywh",
+)
+```
+
+- Through custom configuration controlled by the user. To do this, simply
+  pass the desired configuration parameters to the default constructors of the
+  symbols documented below.
+
+```python
+backbone = keras_cv.models.ResNetBackbone(
+    stackwise_filters=[64, 128, 256, 512],
+    stackwise_blocks=[2, 2, 2, 2],
+    stackwise_strides=[1, 2, 2, 2],
+    include_rescaling=False,
+)
+model = keras_cv.models.RetinaNet(
+    backbone=backbone,
+    num_classes=20,
+    bounding_box_format="xywh",
+)
+```
+
+For a more in depth introduction to how our API fits together, see the
+[getting started guide](guides/keras_cv/).
+
+## Backbone presets
+
+The following preset names correspond to a configuration, and weights
+for a **backbone**. These presets are not inference-ready, and must be
```

Comment:
nit: application-ready. They are definitely ready for inference!

Reply:
The only thing not "inference ready" is a Task initialized from a backbone preset.